### ISSUE TITLE
Underlines in code segments

### ISSUE THREAD

**Issue by yamnikov-oleg** [fabiospampinato/vscode-todo-plus#13]
Hey, Fabio. Thank you for your work, this extension has really boosted my productivity.

To the issue. I like to reference code symbols in my todos like so:

```
[ ] Fix an issue
  Relevant method: SomeClass.method_to_fix
```

But unfortunately formatting makes symbols looks badly if there are underlines in them. In the example above, `to` would be formatted in italics in the middle of method name.

Could you, maybe, make underlines only format text if there's space adjacent to them?

```
[ ] Fix an issue
  some_method_to_fix() -- no italics
  This is _very_ important -- italics
```

---

As a side note I would like to propose code segments formatting with backticks (`). These segments could get slightly different background color and could have other formatting features (stars and underlines) disabled inside them. Though I'm not sure this feature would be useful for anybody but me 😃

**Comment by fabiospampinato** [comment]
Formatting only if there's space or line start/end characters around them seems reasonable. Should be pretty easy to implement.

In the mean time you may disable formatting entirely by adding `"todo.formatting.enabled": false` to your settings.

Code segments may be an interesting addition, but it probably won't be that easy to implement. It would be better if you could open a separate issue for that, so that I can monitor how many people want this.

**Comment by yamnikov-oleg** [comment]
@fabiospampinato done: #14

**Comment by fabiospampinato** [comment]
Fixed in v1.6.0.
